### PR TITLE
fix: update changelog for pending release

### LINKED ISSUE #542: New release perhaps?
Hey, the latest release is almost a year old by now and a few cool things have landed in master. Perhaps it's time for a new release?

### PR DESCRIPTION
**Pull request by rawkode**
Changelog update

cc #542

### CODE REVIEW

**Comment by Emilgardis** [comment]
This does not solve #542, but it touches on it.

**Comment by lpil** [comment]
This looks great! What is required to get this merged and the latest version released? Thank you

**Review by lpil — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,18 @@ This project adheres to [Semantic Versioning](http://semver.org/).
...
+- #552 - Added CHANGELOG.md automation
+- #543 - Added environment variables to control the UID and GID in the container
+- #534 - fix image builds with update of dependencies
+- #502 - fix ci: bump openssl version in freebsd
+- #501 - x86_64-linux: lower glibc version requirement to 2.17 (compatible with centos 7) 
+- #500 - use runner setting specified in Cross.toml 
+- #498 - bump linux-image version to fix CI
+- #497 - don't set RUSTFLAGS in aarch64-musl image 
+- #492 - Add cmake to FreeBSD images
+- #389 - Add support for more hosts and simplify/unify host support checks
```
Comment (adamgreig) [inline CHANGELOG.md:17]:
```suggestion
- #489 - Add support for more hosts and simplify/unify host support checks
```

**Review by adamgreig — CHANGES_REQUESTED** [review]
Thank for putting this together! I spotted one typo which I'll fix then good to merge.

**Review by adamgreig — APPROVED** [review]

**Comment by adamgreig** [comment]
bors merge

**Comment by adamgreig** [comment]
bors merge

:crossed_fingers:

**Pull request merged**